README.md

```diff
@@ -72,3 +75,11 @@ To update a dependency:
 1. Ensure you have govendor installed: `go get -u github.com/kardianos/govendor`
 2. CD to the root of this repo (not into the vendor directory)
 3. Run `govendor fetch PATH/FOR/PACKAGE` where PATH/FOR/PACKAGE is the path property in `vendor.json` of the package you wish to update. Ex. `go get github.com/paultyng/go-newrelic/api`
+
+Finding Certain ID and Key
```

Comment:
This section feels like it's content that should be part of the resource-specific docs that get rendered out at https://www.terraform.io/docs/providers/newrelic/index.html

Reply:
Agreed. Thoughts on adding a hyperlink to this page here to help with discovery?